### ISSUE TITLE
flagging undefined method

### ISSUE THREAD

**Issue by zanona** [sirbrillig/phpcs-variable-analysis#104]
Hi there,
Sorry If I am completely missing the point but is there any way to also allow `phpcs-variable-analysis` to flag undefined method calls, like:

```php
function do_that( $foo, $bar ) { ... }

doThat( 'foo', 'bar' ); //<- throws error as doThat doesn't exist

another_method_with_callback( 'foo', doThat ) // < also throws error as it's calling doThat
```

If not possible, perhaps anyone knows of an alternative?

Thanks in advance

**Comment by sirbrillig** [comment]
That's a good idea. Due to the nature of how PHP passes and imports functions, this can only be done to a limited extent with a linter like this, but we can determine if a function has been imported, defined locally, or used with an absolute path. I made [phpcs-import-detection](https://github.com/sirbrillig/phpcs-import-detection) for that purpose; I hope it helps!

I'm going to close this for now, but if that package doesn't do what you're looking for, we can re-open it.

**Comment by zanona** [comment]
@sirbrillig, thanks so much! That works really well :)
